_(
    import("random"),
    not_random(),
    print("expected error:", catch(random_u16())),
)